src/+matlode/+rk/RungeKutta.m

```diff
@@ -14,6 +14,22 @@
     end
     
     methods
+        function obj = RungeKutta(a, b, bHat, c, e, order, embeddedOrder)
+            
+            obj = obj@matlode.OneStepIntegrator(~isempty(bHat), class(a));
+            
+            obj.A = a;
+            obj.B = b;
+            obj.BHat = bHat;
+            obj.C = c;
+            obj.E = e;
+            obj.EmbeddedOrder = embeddedOrder;
+            obj.Order = order;
+            obj.Stage = size(b, 2);
+            obj.FSAL = all(a(end, :) == b) && all(a(1, :) == 0);
```

Comment:
This relies on equality giving us correct things. If we somehow define things that are almost, but not quite exact, things here may break. There should be a way to set this manually IMO.

Equality with floats is dangerous, as you know.

Reply:
Ah, then a(end,:) - b < eps and a(1, :) < eps should be fine then. I'm not too sure on letting the user do it manually, perhaps an optional parameter?